ddtrace/tracer.py

```diff
@@ -205,6 +213,9 @@ def __init__(
                 sync_mode=self._use_sync_mode(),
                 headers={"Datadog-Client-Computed-Stats": "yes"} if self._compute_stats else {},
             )
+        self._single_span_sampling_rules = (
+            SpanSamplingRules().get_single_span_sampling_rules()
```

Comment:
this is the part where creating an instance of `SpanSamplingRules` feels weird, since we never actually need it to be an instance, we just need a helper to parse the env and give us a list of rules.